_pages/team.html

```diff
@@ -4,8 +4,6 @@
 
 <p>We are the team that brings you the best experience in all things technology.</p>
 
-<p>{% include snippets/join-us.html %}</p>
```

Comment:
You should edit the snippet to be an empty line rather than removing this itself. Also... it's generally bad practice to submit a pull request and then accept the request yourself...

Reply:
Alrighty. I apologise for the pull request bit, I didn't know it was bad practice :P